extension/src/paymentHandler/web-component-service.js

```diff
@@ -0,0 +1,50 @@
+const fetch = require('node-fetch')
```

Comment:
> extension/src/paymentHandler/web-component-service.js

I wonder why this service is nested within paymentHandler package. It is not really a handler, right?

Reply:
You're right.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/232/commits/6e68777b0d3e0f952691b8bdfa46b84ceafd6e20